interactive_engine/src/executor/store/Cargo.toml

```diff
@@ -26,6 +26,7 @@ maxgraph-common = { path = "../../common/rust/common" }
 
 [dependencies.rocksdb]
 git = "https://github.com/pingcap/rust-rocksdb.git"
+rev = "23a604459899f4f7a4c8967f16b6fa623474f27b"
```

Comment:
can we depend on a general/stable verison?

Reply:
This is a historical issue: the MaxGraph storage has depended on this wired version. While it is necessary to migrate to a stable version, I think we can do that by creating another issue.

Reply:
Sure, created as issue #525